src/NuGet/Microsoft.Net.Compilers.netcore.nuspec

```diff
@@ -38,5 +38,33 @@
     <file src="$emptyDirPath$/_._" target="ref/netcoreapp2.0" />
     <file src="Exes/csc/netcoreapp2.0/csc.dll" target="runtimes/any/native" />
     <file src="Exes/vbc/netcoreapp2.0/vbc.dll" target="runtimes/any/native" />
+
+    <!-- Satellite assemblies -->
```

Comment:
Please update Microsoft.NETCore.Compilers.nuspec too. That is the package that gets ingested by https://github.com/dotnet/cli and one of the important outcomes that we need from this work is that it ingest satellites built from this repo instead of the interim https://github.com/dotnet/cli-deps-satellites.

Reply:
This package might not even be used for anything anymore. cc @khyperia